mds/admin.py

```diff
@@ -1,6 +1,16 @@
 from django.contrib import admin
 from . import models
 
+from uuid import UUID
+
+
+def is_uuid(uuid_string, version=4):
+    try:
+        uid = UUID(uuid_string, version=version)
```

Comment:
The fact that UUID could parse it is enough.

Reply:
Apparently, you can by pass the UUID parse check if you replace "-" by other things.

Reply:
I'd like to see an example.

Reply:
My bad :'(